Version 1.2.1 compiuling errors on Pico 2

The recent version 1.2.1. has compiling errors with the standard examples on a Pico 2.
 Problem is not there when switching back to 1.2.0

-------------------
In file included from /Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:47:
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.h: In function 'amy_usbd_init':
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.h:44:5: error: implicit declaration of function 'mp_usbd_init_tud'; did you mean 'amy_usbd_init_tud'? [-Wimplicit-function-declaration]
   44 |     mp_usbd_init_tud();
      |     ^~~~~~~~~~~~~~~~
      |     amy_usbd_init_tud
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c: At top level:
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:62:17: error: 'MICROPY_HW_USB_VID' undeclared here (not in a function)
   62 |     .idVendor = MICROPY_HW_USB_VID,
      |                 ^~~~~~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:63:18: error: 'MICROPY_HW_USB_PID' undeclared here (not in a function)
   63 |     .idProduct = MICROPY_HW_USB_PID,
      |                  ^~~~~~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:65:22: error: 'USBD_STR_MANUF' undeclared here (not in a function)
   65 |     .iManufacturer = USBD_STR_MANUF,
      |                      ^~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:66:17: error: 'USBD_STR_PRODUCT' undeclared here (not in a function); did you mean 'USB_PRODUCT'?
   66 |     .iProduct = USBD_STR_PRODUCT,
      |                 ^~~~~~~~~~~~~~~~
      |                 USB_PRODUCT
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:67:22: error: 'USBD_STR_SERIAL' undeclared here (not in a function)
   67 |     .iSerialNumber = USBD_STR_SERIAL,
      |                      ^~~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c: In function 'tud_descriptor_string_cb':
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:100:21: error: 'MICROPY_HW_USB_DESC_STR_MAX' undeclared (first use in this function)
  100 |     char serial_buf[MICROPY_HW_USB_DESC_STR_MAX + 1]; // Includes terminating NUL byte
      |                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:100:21: note: each undeclared identifier is reported only once for each function it appears in
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:133:28: error: 'MICROPY_HW_USB_MANUFACTURER_STRING' undeclared (first use in this function)
  133 |                 desc_str = MICROPY_HW_USB_MANUFACTURER_STRING;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:136:28: error: 'MICROPY_HW_USB_PRODUCT_FS_STRING' undeclared (first use in this function)
  136 |                 desc_str = MICROPY_HW_USB_PRODUCT_FS_STRING;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:139:18: error: 'USBD_STR_CDC' undeclared (first use in this function)
  139 |             case USBD_STR_CDC:
      |                  ^~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:140:28: error: 'MICROPY_HW_USB_CDC_INTERFACE_STRING' undeclared (first use in this function)
  140 |                 desc_str = MICROPY_HW_USB_CDC_INTERFACE_STRING;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:144:18: error: 'USBD_STR_MSC' undeclared (first use in this function)
  144 |             case USBD_STR_MSC:
      |                  ^~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:145:28: error: 'MICROPY_HW_USB_MSC_INTERFACE_STRING' undeclared (first use in this function)
  145 |                 desc_str = MICROPY_HW_USB_MSC_INTERFACE_STRING;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer/src/usb.c:168:1: error: control reaches end of non-void function [-Werror=return-type]
  168 | }
      | ^
cc1: some warnings being treated as errors
Using library AMY Synthesizer at version 1.2.1 in folder: /Users/mragutli/Documents/Arduino/libraries/AMY_Synthesizer 
Using library I2S at version 2.0 in folder: /Users/mragutli/Library/Arduino15/packages/rp2040/hardware/rp2040/5.5.1/libraries/I2S 
Using library AudioBufferManager at version 1.0.0 in folder: /Users/mragutli/Library/Arduino15/packages/rp2040/hardware/rp2040/5.5.1/libraries/AudioBufferManager 
exit status 1

Compilation error: exit status 1

--- a/src/usb.c
+++ b/src/usb.c
@@ -3,7 +3,7 @@
 
 #include "amy.h"
 
-#if defined AMY_MCU
+#if defined(AMY_MCU) && (!defined(ARDUINO) || defined(AMYBOARD_ARDUINO))
 
 #define CONFIG_TOTAL_LEN_MIDI  (TUD_CONFIG_DESC_LEN + TUD_MIDI_DESC_LEN + TUD_CDC_DESC_LEN)
 

--- a/src/usb.h
+++ b/src/usb.h
@@ -1,7 +1,7 @@
 // usb.h
 // tinyusb stuff
 
-#ifdef AMY_MCU
+#if defined(AMY_MCU) && (!defined(ARDUINO) || defined(AMYBOARD_ARDUINO))
 
 #include "class/audio/audio.h"
 #include "class/midi/midi.h"

